site/src/components/page-sidebar.tsx

```diff
@@ -439,15 +448,8 @@ export const Sidebar: VFC<SidebarProps> = ({
           ref={indicator}
           sx={{
             position: "absolute",
-            width: 3,
-            height: 35,
-            backgroundColor: ({ palette }) => palette.purple[600],
             top: selectedOffsetTop === undefined ? 0 : selectedOffsetTop,
-            opacity: selectedOffsetTop === undefined ? 0 : 1,
-            transition: theme.transitions.create(["top", "opacity"], {
-              duration: 150,
-            }),
-            zIndex: 2,
+            opacity: 0,
```

Comment:
With this approach we lose the indicator's movement animation. It's very subtle anyway, so it's probably not an issue?

Reply:
I think we can have it gone for now - if we want it back we will need the height of this indicator to be variable based on the height of the list item it's next to (and update that through resizes) - not sure it's worth it but will defer to @nathggns for later consideration about whether we update this further (after this PR, which will help with readability)